src/build.d

```diff
@@ -1228,14 +1234,21 @@ void args2Environment(ref string[] args)
 {
     bool tryToAdd(string arg)
     {
-        if (!arg.canFind("="))
+        const parts = arg.findSplit("=");
+
+        if (!parts)
             return false;
 
-        auto sp = arg.splitter("=");
-        auto key = sp.front;
-        auto value = sp.dropOne.front;
-        environment[key] = value;
-        env[key] = value;
+        const key = parts[0];
+        const value = parts[2];
+
+        if (key.among("DFLAGS", "CXXFLAGS")) {
+            flags[key] = value.split();
+        }
```

Comment:
I haven't seen this kind of `if else` structure before, can you normalize it?

Reply:
Thats an old habit, changed to phobos style